security/pkg/pki/testdata/multilevelpki/ecc-certs.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# Copyright Istio Authors
```

Comment:
There is a makefile in one of the sample directory that has useful and similar commands. 

Would be great to use it - I think it's also included in the distro.